chore: add RTL examples for visual testing

mnajdova opened the pull request:
This PR adds mechanism for adding rtl examples for visual test. The added test/examples should end with the extension `.rtl.tsx`, and this is all that is necessary from developer's side (See the example for the `DividerExample.rtl.tsx`). Screener then will maximize this example with rtl enabled and will take screenshot of that.

Commit:
-wip

Commit:
-updated filename

Commit:
-updated changelog

mnajdova commented on docs/src/components/ExternalExampleLayout.tsx:
```diff
@@ -26,23 +27,29 @@ const ExternalExampleLayout: any = props => {
-      </SourceRender.Consumer>
-    </SourceRender>
+    <Provider theme={{ rtl: isRtlEnabled }}>
+      {/*<div dir={isRtlEnabled ? 'rtl' : undefined}>*/}
```
Not sure whether we should add this wrapper div that will have dir='rtl' if rtl is enabled. We have that logic in the ComponentExample component.

layershifter replied:
We definitely need it.

### Without

![image](https://user-images.githubusercontent.com/14183168/51974897-a8a4c180-2489-11e9-84cd-31161978ccc3.png)


### With

![image](https://user-images.githubusercontent.com/14183168/51974922-b8bca100-2489-11e9-99d5-fd6fa0114803.png)

layershifter commented on docs/src/examples/components/Divider/Rtl/DividerExample.rtl.tsx:
```diff
@@ -0,0 +1,6 @@
+
+const DividerExampleShorthand = () => <Divider content="مثال النص" />
+
+export default DividerExampleShorthand
```
```suggestion
export default DividerExampleRtl
```

May be name should be changed?

layershifter committed:
enable div in ExampleLayout, add example with Button

layershifter commented on CHANGELOG.md:
```diff
@@ -20,6 +20,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Fix layout of `Accordion` panel's title @kuzhelov ([#780](https://github.com/stardust-ui/react/pull/780))
 
+### Features
+- Add mechanism for adding rtl visual tests @mnajdova ([#792](https://github.com/stardust-ui/react/pull/792))
```
I'm not sure that we should have an entry in `CHANGELOG` about it, there are no any benefits for user from this change

miroslavstastny reviewed (commented):
I vote for adding RTL examples to docsite as well (and switch them to RTL by default)

Commit:
-added mechanism for popout examples with rtl on or off
-added rtl ecamples for the Button and Divider components
-rtl examples are in rtl mode by default

miroslavstastny commented on docs/src/examples/components/Button/index.tsx:
```diff
@@ -13,6 +14,7 @@ const ButtonExamples = () => (
     <Variations />
     <Groups />
     <Usage />
+    <Rtl />
```
The same change is missing in `Divider/index.tsx`

mnajdova replied:
Added, thanks!

miroslavstastny commented on docs/src/routes.tsx:
```diff
@@ -18,7 +18,7 @@ import IntegrateCustomComponents from './views/IntegrateCustomComponents'
   <BrowserRouter basename={__BASENAME__}>
     <Switch>
-      <Route exact path="/maximize/:exampleName" component={ExternalExampleLayout} />
+      <Route exact path="/maximize/:exampleName/:rtl?" component={ExternalExampleLayout} />
```
I don't like `/true` and `/false` in the URL, but let's leave it as is for now and iterate on that later

miroslavstastny reviewed (commented):
Please also edit `SECTION_ORDER` in `gulp-example-menu.ts` to show RTL below Usage section just above Performance

Commit:
-added Divider example
-changed order in gulp-example-menu

Commit:
Merge branch 'master' into feat/add-rtl-screenshoots-on-screener

Commit:
-fixed changelog after merging master
